modules/gojira-core/src/main/java/com/flipkart/gojira/core/aspects/AOPAllianceAdapter.java

```diff
@@ -50,11 +50,7 @@ public Object around(ProceedingJoinPoint thisJoinPoint) throws Throwable {
         };
         MethodInterceptor mInt = getMethodInterceptor();
         if (mInt != null) {
-            try {
-                return mInt.invoke(mic);
-            } catch (Throwable t) {
-                throw new SoftException(t);
-            }
+            return mInt.invoke(mic);
```

Comment:
Let's remove the import of SoftException as well

Reply:
Done